ops/jenkins/bfd-deploy-apps/Jenkinsfile

```diff
@@ -346,7 +342,7 @@ spec:
                         currentBuild.result = 'FAILURE'
                         throw ex
                     } finally {
-                        sendNotifications(trimmedEnv, currentBuild.currentResult, currentStage, gitCommitId, gitRepoUrl)
+                        //sendNotifications(trimmedEnv, currentBuild.currentResult, currentStage, gitCommitId, gitRepoUrl)
```

Comment:
Assuming this needs adjustment before merge.

Reply:
Yes! Didn't push the last commit. Thank you!